pkgs/development/libraries/pipewire/default.nix

```diff
@@ -27,28 +27,37 @@
 , callPackage
 , nixosTests
 , withMediaSession ? true
-, gstreamerSupport ? true, gst_all_1 ? null
-, ffmpegSupport ? true, ffmpeg ? null
-, bluezSupport ? true, bluez ? null, sbc ? null, libfreeaptx ? null, ldacbt ? null, fdk_aac ? null
+, gstreamerSupport ? true
+, gst_all_1 ? null
```

Comment:
Do we really need `? null` for all optional dependencies? 
They are available by `callPackage` and are lazy evaluated.

Reply:
I'm fairly sure that is just a "that's just how it has always been done" case. At least for me it has, it was that way for existing optional deps so I used the same pattern for the ones I added.